include/cantera/zeroD/ReactorNet.h

```diff
@@ -81,9 +81,21 @@ class ReactorNet : public FuncEval
      */
     void advance(doublereal time);
 
+    /**
+     * Advance the state of all reactors in time. Take as many internal
+     * timesteps as necessary towards *time*, which will be automatically
+     * reduced to stay within an absolute tolerance (if specified).
+     * Returns the time at the end of integration.
+     * @param time Time to advance to (s).
+     */
+    double advanceTowards(double time);
+
     //! Advance the state of all reactors in time.
     double step();
 
+    //! Estimate a future state based on current derivatives.
+    virtual void estimate(double time, int k, double* yest);
```

Comment:
Parameters need to be described.

The usual convention in Cantera for functions which fill in an array that is passed in as an argument is for the name to start with `get`, e.g. `getEstimate`.

Reply:
noted.